doc/stages/filters.sample.rst

```diff
@@ -73,6 +73,20 @@ cell
   Voxel cell size. If ``radius`` is set, ``cell`` is automatically computed
   such that the cell is circumscribed by the sphere defined by ``radius``.
 
+dimension
```

Comment:
Not sure I love this name. Maybe `indicator_dim` or some such? Names are hard.

Reply:
We use this name in `filters.overlay` already, so I'm just following suit.